app/screens/network/Network.js

```diff
@@ -0,0 +1,204 @@
+// @flow
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+import { checkNetworkConnection } from '/redux/network/actions';
+import { SmButton } from '/basicComponents';
+import { smColors } from '/vars';
+import type { Action } from '/types';
+import { communication } from '/assets/images';
+import { shell } from 'electron';
+import { NoNetworkSection, NetworkList } from '/components/network';
+
+const networks: NetworkEntry[] = [
+  {
+    name: 'Test net',
+    color: smColors.orange,
+    status: 'connected'
+  },
+  {
+    name: 'Private net',
+    color: smColors.red,
+    status: null
+  },
+  {
+    name: 'Sky net',
+    color: smColors.lighterBlack,
+    status: null
+  }
+];
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  padding: 60px;
+`;
+
+const Header = styled.span`
+  font-size: 31px;
+  line-height: 42px;
+  font-weight: bold;
+  color: ${smColors.lighterBlack};
+  margin-bottom: 27px;
+`;
+
+const RightPaneHeader = styled.span`
+  font-size: 14px;
+  font-weight: bold;
+  color: ${smColors.darkGray};
+  line-height: 19px;
+  margin-bottom: 35px;
+`;
+
+// $FlowStyledIssue
+const Text = styled.span`
+  font-size: 16px;
+  color: ${({ color }) => `${color || smColors.lighterBlack}`};
+  line-height: 22px;
+`;
+
+const BodyWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+`;
+
+const RightPaneWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+  flex: 1;
+  border: 1px solid ${smColors.borderGray};
+`;
+
+const LeftPaneWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  flex: 2;
+  margin-right: 97px;
+`;
+
+const ImageWrapper = styled.div`
+  width: 100%;
+  max-height: 270px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 50px;
+`;
+
+const Image = styled.img`
+  max-width: 100%;
+  max-height: 100%;
+`;
+
+const ActionLink = styled(Text)`
+  margin-top: 30px;
+  color: ${smColors.green};
+  cursor: pointer;
+  &:hover {
+    opacity: 0.8;
+  }
+  &:active {
+    opacity: 0.6;
+  }
+`;
+
+type NetworkEntry = {
+  name: string,
+  status: ?'loading' | 'connected',
+  color: string
+};
+
+type Props = {
+  checkNetworkConnection: Action,
+  isNetworkConnected: boolean
+};
+
+type State = {
+  networks: NetworkEntry[]
+};
+
+class Network extends Component<Props, State> {
+  timer: any;
+
+  state = {
+    networks: [...networks]
+  };
+
+  render() {
+    const { isNetworkConnected } = this.props;
+    const { networks } = this.state;
+    return (
+      <Wrapper>
+        <Header>Network</Header>
+        <BodyWrapper>
+          <LeftPaneWrapper>
+            {!isNetworkConnected && <NoNetworkSection />}
+            <NetworkList networks={networks} isNetworkConnected={isNetworkConnected} connectToNetwork={this.connectToNetwork} />
```

Comment:
instead of passing "networks" as param change component NetworkList to NetworkEntry and use it as render func for each element in list

Reply:
ok